zerver/lib/event_queue.py

```diff
@@ -453,6 +493,7 @@ def setup_event_queue():
     send_restart_events(immediate=settings.DEVELOPMENT)
 
 def fetch_events(query):
+    # type: (Dict[str, Any]) -> Dict[str, Any]
```

Comment:
Strange... when I switch this to Mapping I get the following error:

```
zerver/tornadoviews.py: note: In function "get_events_backend":
zerver/tornadoviews.py:96: error: Argument 1 to "fetch_events" has incompatible type Dict[str, Any]; expected Mapping[unicode, Any]
```

I would expect you could pass a Dict into an argument annotated as a Mapping, since Mapping is the more generic, duck-typed form of a Dict, right?